src/otx/core/data/transform_libs/mmaction.py

```diff
@@ -70,6 +234,51 @@ def transform(self, results: dict) -> ActionClsDataEntity:
         )
 
 
+@TRANSFORMS.register_module(force=True)
+class PackActionDetInputs(MMPackActionInputs):
```

Comment:
Is it possible to merge `PackActionClsInputs` and `PackActionDetInputs` into `PackActionInputs` as same as 
https://github.com/openvinotoolkit/training_extensions/blob/6f8e3078e12a55fc6375f0d8aba7e91a603e65aa/src/otx/core/data/transform_libs/mmdet.py#L46

We want to minimize the change for the mm* configurations for the better compatibility. It seems that we have to change `PackActionInputs` to `PackActionDetInputs` in the mm* configuration.

Reply:
[d14d790](https://github.com/openvinotoolkit/training_extensions/pull/2747/commits/d14d7905ba067e75b84a1441e2bac302ab299ae8) I merged PackActionClsInputs and PackAcionDetInputs